treeherder/config/settings.py

```diff
@@ -84,14 +82,72 @@
     'treeherder.middleware.FixedHawkResponseMiddleware',
 ] if middleware]
 
-if ENABLE_DEBUG_TOOLBAR:
-    # django-debug-toolbar requires that not only DEBUG be set, but that the request IP
-    # be in Django's INTERNAL_IPS setting. When using Vagrant, requests don't come from localhost:
-    # http://blog.joshcrompton.com/2014/01/how-to-make-django-debug-toolbar-display-when-using-vagrant/
-    # If the Vagrant IPs vary by platform or if there isn't a consistent IP when we switch to Docker,
-    # we'll have to do: https://github.com/jazzband/django-debug-toolbar/pull/805#issuecomment-240976813
-    INTERNAL_IPS = ['127.0.0.1', '10.0.2.2']
+# Templating
+TEMPLATES = [{
+    'BACKEND': 'django.template.backends.django.DjangoTemplates',
+    'APP_DIRS': True,
+}]
+
+# Database
+# The database config is defined using environment variables of form:
+#
+#   'mysql://username:password@host:optional_port/database_name'
+#
+# which django-environ converts into the Django DB settings dict format.
+DATABASES = {
+    'default': env.db_url('DATABASE_URL'),
+}
+
+# We're intentionally not using django-environ's query string options feature,
+# since it hides configuration outside of the repository, plus could lead to
+# drift between environments.
+for alias in DATABASES:
+    # Persist database connections for 5 minutes, to avoid expensive reconnects.
+    DATABASES[alias]['CONN_MAX_AGE'] = 300
+    DATABASES[alias]['OPTIONS'] = {
+        # Override Django's default connection charset of 'utf8', otherwise it's
+        # still not possible to insert non-BMP unicode into utf8mb4 tables.
+        'charset': 'utf8mb4',
+    }
+    if DATABASES[alias]['HOST'] != 'localhost':
+        # Use TLS when connecting to RDS.
+        DATABASES[alias]['OPTIONS']['ssl'] = {
+            'ca': 'deployment/aws/combined-ca-bundle.pem',
+        }
+
+# Caches
+REDIS_URL = env('REDIS_URL')
+if connection_should_use_tls(REDIS_URL):
+    # Connect using TLS on Heroku.
+    REDIS_URL = get_tls_redis_url(REDIS_URL)
 
+CACHES = {
+    'default': {
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': REDIS_URL,
+        'OPTIONS': {
+            # Override the default of no timeout, to avoid connection hangs.
+            'SOCKET_CONNECT_TIMEOUT': 5,
+        },
+    },
+}
+
+# Internationalization
+LANGUAGE_CODE = 'en-us'
```

Comment:
`LANGUAGE_CODE` wasn't set before - it is a new addition in this commit. The Django default value is `en-us` ([docs](https://docs.djangoproject.com/en/1.11/ref/settings/#language-code)) - we've generally avoided repeating any settings in this file that are using the defaults - though open to thoughts about this :-)

Reply:
Django sets this in their [generated settings template](https://github.com/django/django/blob/master/django/conf/project_template/project_name/settings.py-tpl#L106) and IIRC has done for many versions.

That being said!  I think you're right here and I'll remove them :)